usd_qtpy/editor.py

```diff
@@ -94,5 +102,23 @@ def update_panel_checkstate():
 
         panels_menu.aboutToShow.connect(update_panel_checkstate)
 
+        # Render menu
+
+        render_menu = menubar.addMenu("Render")
+        render_labels = "Playblast" , "Snapshot"
```

Comment:
PEP8 - check your linter. :) 
```suggestion
        render_labels = "Playblast", "Snapshot"
```

Reply:
> PEP8 - check your linter. :)

I really should ; _ ;

On the topic of PEP8 on a different note, I know that snake_case is preferable for functions, 
I just look at Qt and Usd and can't help but copy their style to maintain consistency.
That's my reasoning for naming functions the way I do. I will rename them all to be snake case, I will never more be tempted by the demons hiding in libraries.